src/psqlgraph/mocks.py

```diff
@@ -1,565 +1,8 @@
-import abc
-import copy
 import logging
-import random
-import uuid
-from collections import defaultdict, deque
-from typing import Dict, Iterable, List, Optional
 
-import rstr
+from psqlgraph_test_utils import mocks
```

Comment:
How about `from tests.utils import mocks`?
I.e. moving `mocks.py` into `/tests/utils/`?

Reply:
I wonder why git diff is showing so many diffs if the file was just moved?

Reply:
>I wonder why git diff is showing so many diffs if the file was just moved?

cause if you add content to the old file, git will not consider the file is moved.

Reply:
> I thought psqlgraph/mocks.py was going away entirely?

kept until we updated all the imports.

Reply:
The first one is still one project so one artefact but two different directories after installation. One problem with this is that if there is another standalone project with the same name `psqlgraph_mocks`, installing psqlgraph may contaminate it. 
The second one is the python mono repo project structure, its a totally different way of thinking and may require new tooling or changes in ways we use existing tools. A separate project might be easiest.

Reply:
AFAIK, using `pip install psqlgraph[mocks]` would work for a project that needs both, `psqlgraph` and `mocks`.
A project that does not need `mocks`, would just do `pip install psqlgraph`.

Unfortunately, pip folks refused to support it.
https://github.com/pypa/pip/issues/4783